### DISCUSSION TITLE
Are the React MediaStore Hooks still supported?

### DISCUSSION THREAD

**Discussion by andy-stewart-design** [muxinc/media-chrome#909] [General]
I'm unable to import anything from the `media-store.tsx` file without throwing errors. Am I doing something wrong? I see the `media-store.tsx` file in my `node_modules` but I'm unable to access anything from it. For context, importing from the primary `media-chrome/react` package works as expected.

<img width="964" alt="CleanShot 2024-05-14 at 16 34 02@2x" src="https://github.com/muxinc/media-chrome/assets/47165383/6a62977f-4865-47af-b122-1afa800d79c3">

**Comment by cjpillsbury** [comment]
Hey @andy-stewart-design thanks for the interest in media store! Is there any chance you can put together a minimal reproducible example for your issue? There are a few things that could be going on here, some of which may be a bit harder to guarantee across all potential build setups (e.g. see this behemoth 😅 https://github.com/microsoft/TypeScript/issues/52363). Also, just in case, can you:

1. let me know if this was working for you previously and is no longer working (e.g. after a media chrome version update)
2. try the latest version of media chrome if you haven't already
3. try importing using the full path instead of the named export (i.e. `... from 'media-chrome/dist/react/media-store'`). Note that this may fail too, depending on details of your build setup, but it would be good to know.

**Reply by andy-stewart-design** [reply]
Ah, yeah. Switching to the full path (`media-chrome/dist/react/media-store`) fixed it—thank you for the quick reply! When I tried the full path before, I accidentally used `media-chrome/src/js/react/media-store`, which threw for obvious reasons. This package has been overall really nice to work with — thanks providing such a great resource!
